core/bcast/metrics.go

```diff
@@ -26,6 +26,13 @@ var broadcastDelay = promauto.NewHistogramVec(prometheus.HistogramOpts{
 	Buckets:   []float64{.05, .1, .25, .5, 1, 2.5, 5, 10, 20, 30, 60},
 }, []string{"duty"})
 
+var registrationGauge = promauto.NewGaugeVec(prometheus.GaugeOpts{
```

Comment:
Cardinatlity of "slot" is very high, it will break prometheus. 

Suggest:
`core_bcast_recast_registration_slot`

"The slot of the latest validator registration successfully broadcast by pubkey"